packages/openapi-generator/src/sdk-metadata/__snapshots__/sdk-metadata.spec.ts.snap

```diff
@@ -4,50 +4,55 @@ exports[`sdk-metadata generates metadata content for services with pregenerated
 Object {
   "generationAndUsage": Object {
     "apiSpecificUsage": Object {
-      "header": "Sample code for using the client library",
+      "header": "Usage Example",
       "instructions": "import { DummyApi } from '@sap/dummy-package';
 
 const responseData = await DummyApi.DummyFunction().execute({ destinationName:'myDestinationName' });",
-      "text": "In the example below you execute the  <code>DummyFunction</code> of the  <code>DummyApi</code> API.",
+      "text": "To consume the service via the pregenerated typed client library run the code snippet below. For more details about OData client libraries chose \\"OData Consumption Manual\\" from the \\"Helpful Links\\" menu.",
     },
     "generationSteps": Object {
+      "header": "Installation & Generation Steps",
       "instructions": "<ul>
-<li>Download the API specification to your local computer.</li>
-<li>Install the generator <code>npm install -g @sap-cloud-sdk/openapi-generator</code></li>
+<li>Install <a href=\\"https://docs.npmjs.com/downloading-and-installing-node-js-and-npm\\">Node and npm.</a>. The use of the LTS version is preferred.</li>
+<li>Download the API specification from Overview -> API Specification section. Chose JSON format.</li>
+<li>Generate a typed client by running <code>npm install -g @sap-cloud-sdk/openapi-generator</code> to install the code generator.</li>
 <li>Execute the generator <code>openapi-generator --inputDir <inputDirectory> --outputDir <outputDirectory></code></li>
-</ul>
-The steps above will generate a basic TypeScript client. For additional options like transpiling to JavaScript etc. visit our <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/openapi/generate-openapi-client\\">documentation.</a>",
-      "text": "Follow the following generation steps to generate the client.",
+<li>You should now find generated classes in the output directory specified in step 4.</li>
+<li>By default we generate TypeScript only. If you need to transpile it to JavaScript check the <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client#options>\\"code generator CLI parameters in the documentation.</a></li>
+<li>In case of any issues carefully check your configuration and refer to our <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client\\">extended typed client generation manual.</a></li>
```

Comment:
```suggestion
<li>In case of any issues, carefully check your configuration and refer to our <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client\\">extended typed client generation manual</a>.</li>
```